checkov/github/dal.py

```diff
@@ -60,10 +60,17 @@ def _authenticate(self):
         pass
 
     def _request(self, endpoint):
+        if not self.token:
+            return
         url_endpoint = "{}{}".format(self.api_url, endpoint)
         try:
-            return self.http.request("GET", url_endpoint,
-                                     headers=self._headers())
+            request = self.http.request("GET", url_endpoint,
+                                        headers=self._headers())
+            if request.status == 200:
```

Comment:
shouldn't you also catch the non `200` status codes?

Reply:
failing silently